docs/cloud/shared-infrastructure/ops/security.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Security
+dir:
+  text: "Operations"
+  order: 3
+---
+
+### TLS certificates
+
+Kurrent Cloud provisions secure KurrentDB clusters with TLS enabled for HTTP and GRPC using certificates issued by Let's Encrypt, or when Mutual TLS is enabled, a private certificate authority that is unique to each deployment. We automatically renew the certificates before they expire and replace the certificates on all cluster nodes. This is all done with zero impact to client connections or cluster availability.
+
+Third party certificates are not supported.
+
+## Changing the initial password
+
+During the deployment process, Kurrent Cloud will automatically generate an initial password for the `admin` user. This password can be changed via the database UI or API. For more information refer to the [User Authentication](/server/v24.10/security/user-authentication.md#basic-authentication) section of the KurrentDB documentation. Once reset, the `Clear initial admin credentials` button should be clicked on the `Security -> Initial Credentials` tab for the given cluster (refer to the [Admin Credentials](README.md#admin-credentials) section).
```

Comment:
Related to the previous comment, I think it would be helpful to explain that clearing the initial admin credentials means that it can no longer be retrieved from the console.